tests_and_analysis/test/script_tests/test_powder_map.py

```diff
@@ -5,15 +5,21 @@
 
 import pytest
 import numpy.testing as npt
-# Required for mocking
-import matplotlib.pyplot
 from packaging import version
 from scipy import __version__ as scipy_ver
 
 from tests_and_analysis.test.utils import get_data_path, get_castep_path, get_phonopy_path
 from tests_and_analysis.test.script_tests.utils import (
     get_script_test_data_path, get_current_plot_image_data, args_to_key)
-import euphonic.cli.powder_map
+
+pytestmark = pytest.mark.matplotlib
```

Comment:
This code seems to be duplicated a lot. Would it be better to use a module var/func or class property to set this?

Reply:
If imports were done elsewhere, they would need to be passed back to this file so they could be used, I'm not sure that could be done without it being a bit confusing and unconventional. Leaving as is for now.